src/app/events/page.js

```diff
@@ -1,8 +1,18 @@
-import Title from "@/components/Title";
+import Event from "@/components/events/Event";
```

Comment:
go ahead and create a new file src/comopnents/events/Upcoming.js and put that testing code there. Then call Upcoming in the Home and Events page.